tests/tasks/assert_profile_present.yml

```diff
@@ -5,3 +5,8 @@
   assert:
     that: lsr_net_profile_exists
     msg: "profile {{ profile }} does not exist"
+
+- name: "assert that ansible managed comment in '{{ profile }}' is present"
+  assert:
+    that: lsr_net_profile_ansible_managed
```

Comment:
use `lsr_net_profile_ansible_managed | bool` to avoid bare variable problem

Reply:
hmm - but I see the `that` condition above does not use `| bool` either . . . odd that this is working with no warnings . . .

Reply:
Is it ok to add `| bool` to `lsr_net_profile_exists`, too?
```
  assert:
    that: lsr_net_profile_exists
```

Reply:
I would say - do not add `| bool` to either of them, if they are working - I would assume there are more instances of bare variables in `that` conditions - we should investigate why they are working